iphone/Maps/UI/Settings/MWMTTSSettingsViewController.mm

```diff
@@ -242,8 +241,7 @@ - (void)viewDidLoad
     string const savedLanguage = nsSavedLanguage.UTF8String;
     if (savedLanguage != currentBcp47Str && savedLanguage != standart.first &&
         !savedLanguage.empty())
-      m_languages.emplace_back(
-          make_pair(savedLanguage, translatedTwine(bcp47ToTwineLanguage(nsSavedLanguage))));
+      m_languages.emplace_back(make_pair(savedLanguage, translateLocale(savedLanguage)));
```

Comment:
```suggestion
      m_languages.emplace_back(savedLanguage, translateLocale(savedLanguage));
```

Reply:
I really have to look into modern C++. Thanks!